molecule/user_provided/prepare.yml

```diff
@@ -0,0 +1,81 @@
+---
+- name: Prepare
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  become: yes
+  tasks:
+    - name: stopping service
```

Comment:
Couple points here that are not specific to just this task:
* Let's capitalize the task names
* Let's make task names more descriptive, it helps to understand the failure (e.g. "Stop the mariadb service" would be a better name)